### ISSUE TITLE
How to compute loss value?

### ISSUE THREAD

**Issue by tangdori** [pycaret/pycaret#1367]
I'm using Pycaret to perform binary classification tasks.
Is it possible to display "loss value" for each model in the following image?
![classification_compare_models](https://user-images.githubusercontent.com/72227499/122254776-7568fe80-ce9b-11eb-980b-e235c7432617.png)

I wanted to see how loss value changes depending on different parameter settings for each model.

**Comment by pycaret** [comment]
Hi @tangdori 

You can use the `add_metric` function to add Log Loss or any custom metric you would like. I have recently written an article showing the use of custom metrics: https://towardsdatascience.com/predict-customer-churn-the-right-way-using-pycaret-8ba6541608ac

In your case, you wouldn't need to write a metric, you can simply import Log Loss from scikit-learn and pass it in the `add_metric` function. See below example I have created for you on the `juice` dataset:

```
from pycaret.datasets import get_data
data = get_data('juice')

from pycaret.classification import *
s = setup(data, target = 'Purchase', silent = True)

from sklearn.metrics import log_loss
add_metric('logloss', 'Log Loss', log_loss, greater_is_better=False)

best = compare_models()
```
![image](https://user-images.githubusercontent.com/58118658/122257690-3ee0b300-ce9e-11eb-824e-1399368a053b.png)

Hope this helps.
